### ISSUE TITLE
[Regression] My managed AppImages are gone and I can't re-add them

### ISSUE THREAD

**Issue by GeeLeonidas** [mijorus/gearlever#94]
I've just updated Gear Lever to the latest version (1.6.1) and noticed my managed AppImages were gone (from inside the app, not from the menu), also noticing I could'nt re-add them anymore. After some digging and even deleting the app data in order to clear any weird state, it seems to me that the `strings` binary is missing from the sandbox. Here's the traceback when I try to add an AppImage:
```Traceback (most recent call last):
  File "/app/share/gearlever/gearlever/GearleverWindow.py", line 214, in on_drop_event
    return self.on_selected_local_file(list(value))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/gearlever/gearlever/GearleverWindow.py", line 140, in on_selected_local_file
    if len(files) == 1 and self.app_details.set_from_local_file(files[0]):
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/gearlever/gearlever/AppDetails.py", line 114, in set_from_local_file
    list_element = appimage_provider.create_list_element_from_file(file)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/gearlever/gearlever/providers/AppImageProvider.py", line 499, in create_list_element_from_file
    generation=self.get_appimage_generation(file)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/gearlever/gearlever/providers/AppImageProvider.py", line 477, in get_appimage_generation
    check = terminal.sandbox_sh(['strings', '-d', '-n', str(len(detect_v3)), file.get_path()])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/gearlever/gearlever/lib/terminal.py", line 33, in sandbox_sh
    output = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'strings'
```

**Comment by GeeLeonidas** [comment]
Just reproduced this on my laptop, immediately after the update it doesn't print any traceback, but the logs contain the same `No such file or directory: 'strings'` error

**Comment by nerun** [comment]
Exactly the same error here.

**Comment by OlaoluwaM** [comment]
Me too!

**Comment by mijorus** [comment]
I'm sorry guys, I'll release an update asap

**Comment by OlaoluwaM** [comment]
You might already be aware of this @mijorus, but doing some digging,it looks like [this line](https://github.com/mijorus/gearlever/blob/0941e45eb965b644659962107e25405e67a454a8/src/providers/AppImageProvider.py#L477) might be the culprit. Though, I don't know why a binary is being treated like a file path, maybe an issue with python subprocesses, idk

**Comment by mijorus** [comment]
It was working just fine on my system btw

**Comment by OlaoluwaM** [comment]
> It was working just fine on my system btw

As a flatpak?

**Comment by mijorus** [comment]
Yeah

**Comment by nerun** [comment]
For me no, not working.

**Comment by nerun** [comment]
Fixed with 1.6.2, working now.

**Comment by mijorus** [comment]
Thanks for contributing with this bug report.

If you want to support my work, please consider a small donation.

https://ko-fi.com/mijorus
